### DISCUSSION TITLE
UI suggestion: play folder

### DISCUSSION THREAD

**Discussion by user9931** [jcorporation/myMPD#744] [Ideas]
Thank you for great app! Its not a bug request, but a little ui change to consider...
I'm using filesystem view (not artist/album) because folder/files structure are simple to organize and with some kind of music (classical for example) its almost impossible, so...

Its wold be very helpful to have a button to play whole folder; now its requre to open filder -> click on cue file -> click on track OR there's a button on to select action with "append/replace/add to homescreen" "directory" menu, but a simple "play" or button with configured action (in advanced settings, like "Click album play button", "Click playlist" and other) can be great.

_Option to configure "Click folder" is a little confising... and i'm dont really understand how it can be used... when its changed to something other then "open folder", its completely block folder navigation..._

**Comment by jcorporation** [comment]
> Option to configure "Click folder" is a little confising... and i'm dont really understand how it can be used... when its changed to something other then "open folder", its completely block folder navigation...

Right, this option makes no sense, if there is no other way to open the folder. I thought there was an open folder entry in the context menu but it isn't.

**Reply by jcorporation** [reply]
Thinking about this suggestion: It makes sense to remove the default action for folders and add a quick playback default action in settings. Displaying a play button near by the action button makes sense for pretty all table views.

#749
